lib/webenginesettings.cpp

```diff
@@ -178,6 +186,9 @@ void SailfishOS::WebEngineSettings::initialize()
                                   QVariant::fromValue<bool>(true));
```

Comment:
This comment relates to lines 163-172 (but it won't allow me to put a comment there).

I've not tried this out, but my understanding of the code is that the config values will only be forcefully set on first execution; after which they act like normal config values.

Most of the values are static and never change (except between devices). However the locale may change between browser runs. So I'm thinking maybe the code to set the `Accept-Language` header should sit outside the marker file guard?

Reply:
Good catch, will move `Accept-Language` so that it will be executed always

Reply:
Fixed